portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,29 +36,25 @@
 
   private List<String> messages = new ArrayList<>();
   private static final String SPACE = " ";
+  private String numberChoiceString = "";
 
   /** Get all comment entities in the datastore and add them to messages. */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    System.out.println("DO GET");
```

Comment:
Are these for general logging? If so, we could try to use the logging frameworks that Google Cloud Platform provides: https://cloud.google.com/run/docs/logging#container-logs

Having said that, it's more of a bonus and can be done in a follow up pull request.

Reply:
+1, can simply delete all these prints that are used for debugging.

Reply:
My print statements were for the purpose of debugging. I haven't heard about logging before, but I just read up a bit on it. For now, I'll delete the print statements.